araproc/analysis/spark_power_ratio.py

```diff
@@ -0,0 +1,102 @@
+# this looks for and identifies the spark events by calculating power ratio
+# and comparing with a set threshold 
+
+import numpy as np
+
+from araproc.framework import waveform_utilities as wfu
+
+
+
+
+
+def get_rms_power(waveform):
```

Comment:
Is this the rms power? Maybe its more like the mean power?

Reply:
I call it RMS power because you get this technically by squaring the RMS voltage. But yes, this is basically mean power. I can change the name if required!

Reply:
Hm, but then its actually squared rms voltage? Which is the mean power not the rms power? I think we should just make sure the name reflects what's actually calculated so people don't grab the function thinking it calculates one thing when it's calculating another.